build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     //Unit testing framework.
     testCompile 'junit:junit:4.12'
+    //testCompile "org.mockito:mockito-core:2.+"
```

Comment:
What's up with this commented-out line that got added? Is it supposed do be used somehow?